compiler/luci/pass/src/helpers/HuffmanDecoder.h

```diff
@@ -0,0 +1,355 @@
+/*
```

Comment:
this is header only file. 
- is there any particular reason to make so? why not split implementations to .cpp file?
- copy right contains only Samsung. is this file made by you from scratch?
- there is no .test.cpp file for this. can you add some?

Reply:
1. ok
2. yes
3. ok